@screen and @apply (#115)
tw-in-js/twind: docs/getting-started/css-in-js.md

`
```

> As a convenience Twind provides the `@apply` CSS rule:

Reviewer: ```suggestion
For convenience, Twind provides the `@apply` CSS rule:
```

```js
css`
  @apply text-gray(700 dark:300);

  p {
    @apply my-5;
  }

  h1 {
    @apply text(black dark:white hover:purple-500);
  }
`

css`
  @apply ${['font-bold', false && 'underline', 'py-2 px-4']};
  color: fuchsia;
`
```

> `@apply can be use in the object notation as well:

Reviewer: ```suggestion
`@apply can be used in the object notation as well:
```

`
```

> As a convenience Twind provides the `@screen` CSS rule:

Reviewer: ```suggestion
For convenience, Twind provides the `@screen` CSS rule:
```
